sigle/src/modules/layout/components/AppHeader.tsx

```diff
@@ -0,0 +1,135 @@
+import { EyeOpenIcon as EyeOpenIconBase } from '@radix-ui/react-icons';
+import Link from 'next/link';
+import Image from 'next/image';
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+import { toast } from 'react-toastify';
+import { styled } from '../../../stitches.config';
+import { Button, Container, Flex, Text } from '../../../ui';
+import {
+  convertStoryToSubsetStory,
+  createNewEmptyStory,
+  getStoriesFile,
+  saveStoriesFile,
+  saveStoryFile,
+} from '../../../utils';
+import * as Fathom from 'fathom-client';
+import { useAuth } from '../../auth/AuthContext';
+import { Goals } from '../../../utils/fathom';
+
+const Header = styled('header', Container, {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  mt: '$4',
+
+  '@md': {
+    mt: '$10',
+  },
+});
+
+const Logo = () => (
```

Comment:
as the component is really simple, it might be better to just inline it in the code, wdyt?